src/encoded/tests/test_renderers.py

```diff
@@ -0,0 +1,128 @@
+import pytest
```

Comment:
This test file is new and is a copy of what `cgap-portal` does. The tests even in CGAP are new but at least help to notice regressions in the mime stuff.